<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>bracingScheme</class>
 <widget class="QDialog" name="bracingScheme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="editBracingSchemeLabel">
             <property name="font">
              <font>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Edit Bracing Scheme</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="addBracingSchemeButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QToolButton" name="deleteBracingSchemeButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QTableWidget" name="bracingSchemeTable">
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustIgnored</enum>
           </property>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <column>
            <property name="text">
             <string>Bracing</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="ViewSectionWidget" name="bracingSchemeViewer" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>1</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>250</width>
           <height>0</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0,0,0,0">
         <property name="spacing">
          <number>6</number>
         </property>
         <item>
          <widget class="QLabel" name="defineBracingLabel">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Define Bracing Scheme</string>
           </property>
           <property name="margin">
            <number>7</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="bracingNameLabel">
             <property name="text">
              <string>Name</string>
             </property>
             <property name="margin">
              <number>5</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="bracingNameEdit"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="bracingCoordLabel">
           <property name="text">
            <string>X Y Coordinates</string>
           </property>
           <property name="margin">
            <number>5</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="bracingCoordTable">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="maximumSize">
            <size>
             <width>350</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="contextMenuPolicy">
            <enum>Qt::DefaultContextMenu</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAsNeeded</enum>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QAbstractScrollArea::AdjustToContents</enum>
           </property>
           <property name="horizontalScrollMode">
            <enum>QAbstractItemView::ScrollPerItem</enum>
           </property>
           <column>
            <property name="text">
             <string>x1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>y1</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>x2</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>y2</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="bracingSchemeButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ViewSectionWidget</class>
   <extends>QWidget</extends>
   <header location="global">View2DEngine</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
